scene/gui/tree.cpp

```diff
@@ -37,6 +37,7 @@
 #include "core/os/os.h"
 #include "core/string/print_string.h"
 #include "core/string/translation.h"
+#include "core/variant/variant.h"
```

Comment:
Why was this added? This is already included, or was there some issue?

Reply:
It is not necessary. It was an auto inserted by the IDE at some point and I didn't realize. I recompiled and it worked just fine.